usecases/schema/property.go

```diff
@@ -68,44 +68,12 @@ func (h *Handler) AddClassProperty(ctx context.Context, principal *models.Princi
 
 	// TODO-RAFT use UpdateProperty() for adding/merging property when index idempotence exists
 	// revisit when index idempotence exists and/or allowing merging properties on index.
-	new, old := split(class.Properties, newProps)
-	if len(old) > 0 {
-		mergeClassExistedProp(class, old...)
-		if _, err = h.metaWriter.UpdateClass(class, nil); err != nil {
-			return 0, err
-		}
-	}
-
-	if len(new) == 0 {
+	props := schema.DedupProperties(class.Properties, newProps)
```

Comment:
**question:** shall that happen before `migratePropertySettings` L67 ?

Reply:
nice suggestion. changed